pkg/controller/user/create.go

```diff
@@ -96,17 +98,35 @@ func (c *Controller) HandleCreate() http.Handler {
 			return
 		}
 
+		if !alreadyExists {
+			pwd, err := otp.GenerateAlphanumericCode(24)
```

Comment:
We should use [go-password](https://github.com/sethvargo/go-password) or [go-diceware](https://github.com/sethvargo/go-diceware) here instead since these are use passwords.